--- share.txt	2020-06-04 02:41:45.542231589 +0200
+++ /usr/share/X11/xkb/symbols/pc	2020-06-04 02:47:54.215553186 +0200
@@ -19,16 +19,16 @@
     key  <TAB> {	[ Tab,	ISO_Left_Tab	]	};
     key <RTRN> {	[ Return		]	};
 
-    key <CAPS> {	[ Caps_Lock		]	};
+    key <CAPS> {	[ BackSpace, BackSpace	]	};
     key <NMLK> {	[ Num_Lock 		]	};
 
     key <LFSH> {	[ Shift_L		]	};
-    key <LCTL> {	[ Control_L		]	};
-    key <LWIN> {	[ Super_L		]	};
+    key <LCTL> {	[ Super_L		]	};
+    key <LWIN> {	[ Control_L		]	};
 
     key <RTSH> {	[ Shift_R		]	};
-    key <RCTL> {	[ Control_R		]	};
-    key <RWIN> {	[ Super_R		]	};
+    key <RCTL> {	[ Super_R		]	};
+    key <RWIN> {	[ Control_R		]	};
     key <MENU> {	[ Menu			]	};
 
     // Beginning of modifier mappings.
